### PR TITLE
gh-119580: Improve version added section for convenience variable

### PR DESCRIPTION
**Pull request by gaogaotiantian**
The version added section is not clear about what is "added" because there are a lot of content above. Clarify it by explicitly write the feature.

* Issue: gh-119580

----
📚 Documentation preview 📚: https://cpython-previews--119583.org.readthedocs.build/

### CODE REVIEW

File: Doc/library/pdb.rst [hunk 1]
```diff
@@ -288,7 +288,7 @@ There are three preset *convenience variables*:
 * ``$_retval``: the return value if the frame is returning
 * ``$_exception``: the exception if the frame is raising an exception
 
-.. versionadded:: 3.12
+.. versionadded:: 3.12  *convenience variable*
```
Comment (AlexWaygood) [inline Doc/library/pdb.rst]: I think this is how we'd normally do this kind of thing:

```suggestion
.. versionadded:: 3.12

   Added the three convenience variables
```

E.g. see https://github.com/python/cpython/blob/5d04cc50e51cb262ee189a6ef0e79f4b372d1583/Doc/library/typing.rst?plain=1#L891-L897

Comment (gaogaotiantian) [reply Doc/library/pdb.rst]: Okay I copied the format from `enum.rst` where they do thing differently...

Comment (gaogaotiantian) [reply Doc/library/pdb.rst]: Also this is what confused the user - we did not only add the three convenience variables, we added the convenience variable feature, together with three pre-defined convenience variables.

Comment (AlexWaygood) [reply Doc/library/pdb.rst]: > Okay I copied the format from `enum.rst` where they do thing differently...

Oh, fair enough!

> Also this is what confused the user - we did not only add the three convenience variables, we added the convenience variable feature, together with three pre-defined convenience variables.

I see. I think your proposed text is maybe slightly too terse in that case; I still misread it as a non-pdb expert

File: Doc/library/pdb.rst [hunk 2]
```diff
@@ -289,6 +289,7 @@ There are three preset *convenience variables*:
 * ``$_exception``: the exception if the frame is raising an exception
 
 .. versionadded:: 3.12
+   *convenience variable*
```
Comment (AlexWaygood) [inline Doc/library/pdb.rst]:
```suggestion

   Added the *convenience variable* feature.
```

Comment (gaogaotiantian) [reply Doc/library/pdb.rst]: Yeah that makes sense.

**Review by AlexWaygood — APPROVED** [review]
Thanks!

**Pull request merged**